databases/Akumuli/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        akumuli Akumuli 0.8.78 v
+categories          databases
+platforms           darwin
+maintainers         nomaintainer
+license             Apache-2
+
+description         Akumuli is a time-series database for modern hardware
+long_description    Akumuli is a time-series database for modern hardware. It can be used to capture, \
+                    store and process time-series data in real-time. \
+                    The word akumuli can be translated from Esperanto as accumulate.
+
+checksums           sha256  110fff1c2c636f5361d5ca8d510e99c1e55af717c9731343066299d1d435bf8c \
+                    rmd160  e4cdaf87f55ffdd369a71cd7c519c6c24bcf6f14 \
+                    size    551148
+
+depends_lib-append  port:boost \
+                    port:jemalloc \
+                    port:sqlite3 \
+                    port:libmicrohttpd \
+                    port:apr \
+                    port:apr-util \
+                    port:muparser \
+                    port:log4cxx
```

Comment:
the build system doesn't find `log4cxx` and `muparser` even though you've added the dependencies here:
`-- NOT Found log4cxx, disabling it`
`-- Not found muParser`